tests/backup/backup_test_labels.go

```diff
@@ -510,4 +512,5 @@ var TestCaseLabelsMap = map[TestCaseName][]TestCaseLabel{
 	BackupStateTransitionForScheduledBackups:                         {BackupStateTransitionForScheduledBackupsLabel},
 	EnableNsAndClusterLevelPSAWithBackupAndRestore:                   {EnableNsAndClusterLevelPSAWithBackupAndRestoreLabel},
 	RestoreFromHigherPrivilegedNamespaceToLower:                      {RestoreFromHigherPrivilegedNamespaceToLowerLabel, rkePipelineNightly},
+	PSALowerPrivilegeToHigherPrivilegeWithProjectMapping:             {PSALowerPrivilegeToHigherPrivilegeWithProjectMappingLabel},
```

Comment:
can add label rkePipelineNightly since it can run only on RKE

Reply:
Done